packages/wrangler/src/parse.ts

```diff
@@ -359,3 +359,39 @@ export function parseNonHyphenedUuid(uuid: string | null): string | null {
 
 	return hyphenated.slice(0, 36);
 }
+
+export function parseByteSize(
```

Comment:
I think that this function doesn't correctly parse values that incorrectly end with just `ib`
![Screenshot 2025-05-23 at 15 35 38](https://github.com/user-attachments/assets/55d9538d-89a5-4010-b2ec-bf2a22d166dc)

shouldn't these result in `NaN`s?

Reply:
Good catch, thanks! Updated the function and added some test cases.